### PR TITLE
Admin: Add delete for service providers admin module (SHOOP-2297)

### PR DESCRIPTION
**Pull request by tulimaki**
Services linked to service provider is not removed, but the
service provider foreign key is set to null and service is disabled.

Refs SHOOP-2297

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -49,6 +49,7 @@ Localization
 Admin
 ~~~~~
 
+- Add delete for service providers admin module
```
Comment (shawnadelic) [inline doc/changelog.rst]: Is this needed since we only just added the service providers module?

Comment (suutari-ai) [inline doc/changelog.rst]: Yep, this changelog entry should be removed from this PR.

**Comment by suutari-ai** [comment]
When the changelog entry is removed, this is good to go.

**Comment by suutari-ai** [comment]
I removed the changelog entry (and added entry for service provider admin) and merged this with that change.

Thanks guys!

**Pull request merged**